<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZQD</class>
 <widget class="QMainWindow" name="ZQD">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>672</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>254</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>399</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;Arial&quot;;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_mod">
           <property name="minimumSize">
            <size>
             <width>93</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>61</height>
            </size>
           </property>
           <property name="text">
            <string>%</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ce">
           <property name="minimumSize">
            <size>
             <width>93</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>61</height>
            </size>
           </property>
           <property name="text">
            <string>CE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear">
           <property name="minimumSize">
            <size>
             <width>93</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>61</height>
            </size>
           </property>
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_backspace">
           <property name="minimumSize">
            <size>
             <width>93</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>93</width>
             <height>61</height>
            </size>
           </property>
           <property name="text">
            <string>‚Üê</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_1_div_x">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>1/X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_x_pow">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>X^2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sqrt_x">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>Sqrt(x)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_divide">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_multiply">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minus">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton_1">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_reverse">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_0">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_point">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_equal">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>61</height>
          </size>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
